src/NotificationService.Models.Dto/NotificationService.Models.Dto.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.53" />
+    <PackageReference Include="LT.DigitalOffice.Kernel.BrokerSupport" Version="1.0.1" />
+    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.56" />
+  </ItemGroup>
```

Comment:
maybe remove this and add a connection to Common.Configurations.targets?
same with other .csproj files